docs/tutorials/experimental-fits.md

```diff
@@ -0,0 +1,436 @@
+Load the data
+
+```python
+def load(file):
+    x = np.loadtxt(file)
+    x = jnp.array(x)
+    return x
+
+data = load("short.npy")
+time = load("time_short.npy")
+alpha2 = data[0]
+```
+
+and plot them
+
+```python
+plt.figure(figsize=(10, 5))
+
+plt.plot(time, data, "+")
+plt.grid()
+plt.xlabel("Time [us]")
+plt.ylabel("Photon number")
+renderfig('g2-real-data')
+```
+
+% invisible-code-block: python
+% exp_time, exp_data, exp_alpha2 = time, data, alpha2
+
+![Loaded data](/figs-docs/g2-real-data.png)
+
+#### Option 2: Synthetic data
+
+Create the synthetic data 
+
+```python
+g2_true = 1.2 * MHz
+b0 = dq.unit(dq.fock(Nb, 0) + thermal_b ** 0.5 * dq.fock(Nb, 1))
+
+alpha2 = np.linspace(1, 3, 3)
+rho0 = jnp.stack([dq.tensor(dq.coherent(Na, alpha), b0) for alpha in alpha2 ** 0.5])
+
+H = g2_true * a @ a @ dq.dag(b)
+H += dq.dag(H)
+
+La_down = np.sqrt(kappa_a * (1 + thermal_a)) * a
+La_up   = np.sqrt(kappa_a * thermal_a) * dq.dag(a)
+Lb_down = np.sqrt(kappa_b * (1 + thermal_b)) * b
+Lb_up   = np.sqrt(kappa_b * thermal_b) * dq.dag(b)
+
+time = jnp.linspace(0, 0.6, 501)
+result = dq.mesolve(
+    H, [Lb_down, Lb_up, La_down, La_up],
+    rho0, time,
+    exp_ops=[dq.dag(a) @ a],
+    solver=dq.solver.Dopri5()
+)
+
+data = result.expects[:, 0].real.T
+key = jax.random.key(42)
+data += (jax.random.uniform(key, data.shape) - 0.5) * 0.15
+```
+and plot them
+```python
+plt.figure(figsize=(10, 5))
+
+plt.plot(time, data, "+")
+plt.grid()
+plt.xlabel("Time [us]")
+plt.ylabel("Photon number")
+renderfig('g2-synthetic-data')
+```
+![Synthetic data](/figs-docs/g2-synthetic-data.png)
+
+
+### Perform the fit
+
+% invisible-code-block: python
+% time, data, alpha2 = exp_time, exp_data, exp_alpha2
+
+We define the simulation code
+
+```python
+def simulate(g2):
+    b0 = dq.unit(dq.fock(Nb, 0) + thermal_b ** 0.5 * dq.fock(Nb, 1))
+    rho0 = jnp.stack([dq.tensor(dq.coherent(Na, alpha), b0) for alpha in alpha2 ** 0.5])
+
+    H = g2 * a @ a @ dq.dag(b)
+    H += dq.dag(H)
+
+    La_down = np.sqrt(kappa_a * (1 + thermal_a)) * a
+    La_up   = np.sqrt(kappa_a * thermal_a) * dq.dag(a)
+    Lb_down = np.sqrt(kappa_b * (1 + thermal_b)) * b
+    Lb_up   = np.sqrt(kappa_b * thermal_b) * dq.dag(b)
+
+    result = dq.mesolve(
+        H, [Lb_down, Lb_up, La_down, La_up],
+        rho0, time,
+        exp_ops=[dq.dag(a) @ a],
+    )
+
+    return result
+```
+
+We choose an initial guess that is reasonable with respect to the data we have. We converge to a steady state in roughly $1.0\mu$s so $g_2$ should be of the order of $1$MHz. Let's take $0.1$MHz to have some margin.
+```python
+g2 = jnp.array(0.1 * MHz)
+result = simulate(g2)
+
+plt.figure(figsize=(10, 5))
+
+time_np = time
+for i in range(len(alpha2)):
+    plt.plot(time_np, data[:, i].real, f"+C{i}")
+    plt.plot(time_np, result.expects[i, 0].real, f"C{i}")
+plt.grid()
+plt.xlabel("Time [us]")
+plt.ylabel("Photon number")
+renderfig('g2-initial-guess')
+```
+![g2 initial guess](/figs-docs/g2-initial-guess.png)
```

Comment:
This figure looks a bit silly IMO. Maybe taking $g_2$ to be much larger (say 5 MHz) would be better?

Reply:
It makes sense to show that the fit is bad in a first place. At 5MHz you have a batch of oscillations, it feels like a deliberately bad choice

Reply:
![Uploading image.png…]()

Reply:
I think it looks better, we clearly see it's not the right value no?

Reply:
We also see it with the low value, and it's more complicated  to me. It makes more sense to start from lower values because it entails less non linearities in you system so it's easier to understand